book/templating.rst

```diff
@@ -1222,12 +1222,6 @@ automatically:
             <p>Application Environment: <?php echo $app->getEnvironment() ?></p>
         <?php endif ?>
 
-.. versionadded:: 2.6
-    The global ``app.security`` variable (or the ``$app->getSecurity()``
-    method in PHP templates) is deprecated as of Symfony 2.6. Use ``app.user`` 
-    (``$app->getUser()``) and ``is_granted()`` (``$view['security']->isGranted()``)
-    instead.
```

Comment:
`app.security` isn't used here, so it doesn't make sense to have this tip box here.